DNN Platform/Library/Entities/Profile/ProfileController.cs

```diff
@@ -602,11 +602,11 @@ private static string RemoveUnsafeAttributes(string content)
                 needEncode = true;
             }
 
-            var tagMatches = Regex.Matches(cleanContent, @"(<[^>]*?) on.*?\=(['""]?)[\s\S]*?(\2 )([^>]*?>)", RegexOptions.Compiled | RegexOptions.IgnoreCase);
+            var tagMatches = Regex.Matches(cleanContent, @"(<[^>]*?) on.*?\=(['""]?)[\s\S]*?(\2 )([^>]*?>)", RegexOptions.IgnoreCase);
```

Comment:
For performance purposes, these RegEx statements should be compiled to ensure that they are properly cached and not re-interpreted by the .NET framework.  As only the "most commonly used" items are cached if not explicitly marked as compiled.